source/include/Date.h

```diff
@@ -43,13 +43,15 @@ class Date {
         Int32 _daylightSavingTime;
         static Int32 _systemLocaleTimeZone;
 
-    public:
-        Date(Timestamp timestamp, Int32 timeZoneOffset);
-        ~Date();
-        static void getDate(Timestamp timestamp, Int64* secondsOfYearPtr, Int32* yearPtr,
+    private:
+        void getDate(Timestamp timestamp, Int64* secondsOfYearPtr, Int32* yearPtr,
             Int32* monthPtr = NULL, Int32* dayPtr = NULL, Int32* hourPtr = NULL,
             Int32* minutePtr = NULL, Int32* secondPtr = NULL, Double* fractionPtr = NULL,
             Int32* weekPtr = NULL, Int32* weekOfFirstDay = NULL, char** timeZoneString = NULL);
+
+    public:
+        Date(Timestamp timestamp, Int32 timeZoneOffset);
+        ~Date();
```

Comment:
It looks like you are making the existing ~Date() to fail. I suspect you have not pulled from ni/incoming recently. I think you should pull and merge your changes on the latest.